service/src/main/java/bio/terra/tanagra/app/controller/CohortsApiController.java

```diff
@@ -195,36 +195,36 @@ public ResponseEntity<ApiInstanceCountList> queryCohortCounts(
 
   private static CohortRevision.CriteriaGroupSection fromApiObject(ApiCriteriaGroupSection apiObj) {
     BooleanAndOrFilter.LogicalOperator operator;
-    JoinOperator joinOperator;
-    switch (apiObj.getOperator()) {
-      case OR:
-        operator = BooleanAndOrFilter.LogicalOperator.OR;
-        joinOperator = null;
-        break;
-      case AND:
-        operator = BooleanAndOrFilter.LogicalOperator.AND;
-        joinOperator = null;
-        break;
-      case DURING_SAME_ENCOUNTER:
-        operator = BooleanAndOrFilter.LogicalOperator.OR;
-        joinOperator = JoinOperator.DURING_SAME_ENCOUNTER;
-        break;
-      case WITHIN_NUM_DAYS:
-        operator = BooleanAndOrFilter.LogicalOperator.OR;
-        joinOperator = JoinOperator.WITHIN_NUM_DAYS;
-        break;
-      case NUM_DAYS_BEFORE:
-        operator = BooleanAndOrFilter.LogicalOperator.OR;
-        joinOperator = JoinOperator.NUM_DAYS_BEFORE;
-        break;
-      case NUM_DAYS_AFTER:
-        operator = BooleanAndOrFilter.LogicalOperator.OR;
-        joinOperator = JoinOperator.NUM_DAYS_AFTER;
-        break;
-      default:
-        throw new IllegalArgumentException(
-            "Unknown criteria group section operator: " + apiObj.getOperator());
-    }
+    JoinOperator joinOperator =
```

Comment:
I don't think this particular conversion is an upgrade. It's not shorter and now there's two different styles instead of one.

Reply:
from my understanding it drills down to how it is compiled for execution. was introduced in java 17

Reply:
I'm not sure what you mean. We're not worried about the performance or anything like that here so how it compiles isn't important. In general, the new syntax cleaner and has less boilerplate in a lot of cases. Most of the replacements in the PR are a clear improvement. In this particular case though, because of the two variables are being assigned, changing the way one is assigned just makes it less clear the behavior for both variables is actually the same.

Reply:
Not highly opinionated on it, and on second look agree with you - so no concern reverting it. 
Adding this revert to https://github.com/DataBiosphere/tanagra/pull/1001